### DISCUSSION TITLE
Help getting a new e-ink display device Zigbee2MQTT supported

### DISCUSSION THREAD

**Discussion by Nitramevo** [Koenkk/zigbee2mqtt#17149] [Devices]
**If this is in anyway inappropriate and in violation of any discussion rules, please excuse me and I will delete this post.**

I posted in the global HA FB group and was encouraged to post here.

[Our device is currently only certified for EAA countries: the 27 EU member states and 3 EFTA countries, and UK: until 31 December 2024, where UKCA marking become mandatory].

We are a small boot-strapped start-up that has developed a Zigbee battery-powered e-ink display, which currently exclusively supports the Homey platform: [https://smplproducts.com/](https://smplproducts.com/?fbclid=IwAR1P2K3NbC7hjIU8To651-c7xwWe_7-dW43_O2FS6mtzQEvpkhWFRG1ksm8).

But we get quite a few inquiries from HA, node-red users etc. and are looking for interested parties who will help integrate broader utilizing Zigbee2MQTT.

We can only deliver our product as payment.

Martin...

**Comment by oliveres** [comment]
It will be great when a finished product with an e-ink display and open interface is available on the market. All I've found so far have been just displays, without a box, or with updating manually via NFC. Such products are not suitable for larger deployments. I'm happy to contribute to this.

**Reply by Nitramevo** [reply]
That's just great! Big thx @oliveres 
I need you to sign a 1-page NDA before sending stuff to you. Please send an e-mail to martin@martinstechgarage.com and I will revert with misc.
It would also be great if you could describe in the e-mail how/what you see as next steps (what do you need). And your contact details/address etc. for shipping a couple of displays to you.
Martin...

**Comment by junosuarez** [comment]
If you're looking for additional help, I'd be happy to as well. I'm currently using similar DIY devices made with esp32 and waveshare epaper displayed, but having a commercial product with an attractive case would be great! From taking a quick look at your website, much of what you're hoping to achieve might be necessary within the existing zigbee2mqtt webUI interface by creating a zigbee-herdsman-converter file defining the settings. What's a little unusual is that zigbee2mqtt would be setting periodic data *to* device, instead of receiving reports *from* the device - but this is still possible.

**Reply by Nitramevo** [reply]
Yes please.
We haven't progressed yet.

I need you to sign a 1-page NDA before sending stuff to you. Please send an e-mail to martin@martinstechgarage.com and I will revert with misc.
It would also be great if you could describe in the e-mail how/what you see as next steps (what do you need). And your contact details/address etc. for shipping a couple of displays to you.

Martin...

**Reply by qorron** [reply]
Has this progressed in any way? I'm looking for battery powered zigbee controlled e-ink display such as this.
